tests/dialects/test_arith.py

```diff
@@ -116,9 +117,22 @@ def test_arith_ops(self, func: type[BinaryOperation[_BinOpArgT]]):
         assert op.operands[1].owner is self.b
 
 
+def test_select_op():
+    t = Constant.from_int_and_width(1, IntegerType(1))
+    f = Constant.from_int_and_width(0, IntegerType(1))
+    select_t_op = Select(t, t, f)
+    select_f_op = Select(f, t, f)
+    select_t_op.verify_()
+    select_f_op.verify_()
+
+    # wanting to verify it actually selected the correct operand, but not sure if in correct scope
+    assert select_t_op.result.type == t.result.type
+    assert select_f_op.result.type == f.result.type
```

Comment:
Couldn't figure out how to connect operand to the result and I saw the self verify already does this assert